content/tracing/product_specs/_index.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Product Specs
+kind: documentation
+---
+
+{{< whatsnext desc="Enclosed in this section are other articles that may be of interest:">}}
+    {{< nextlink href="/api/?lang=python#tracing" >}} Learn more about our Trace API {{< /nextlink >}}
```

Comment:
I find it jarring that this links out of this section.  There's already a link in the sidebar nav right below "Product Specs" so let's just take this off the list.  Same for monitors and dashboards.

Reply:
👌